src/index.js

```diff
@@ -1,14 +1,27 @@
 const beginFloat = '~begin~float~';
 const endFloat = '~end~float~';
 
-const StringifyWithFloats = (config = {}) => (value, options) => {
-  const jsonReplacer = (key, value) => {
-    const shouldBeReplaced = config[key] === 'float'
+const StringifyWithFloats = (config = {}) => (inputValue, inputReplacer, space) => {
+  let isFirstIteration = true;
+  const jsonReplacer = (key, val) => {
+    if (isFirstIteration) {
+      isFirstIteration = false;
+      return (typeof inputReplacer === 'function') ? inputReplacer(key, val) : val;
+    }
+    let value;
+    if (typeof inputReplacer === 'function') {
```

Comment:
Can we abstract `typeof inputReplacer === 'function'` up since its used here and in line 9?